Author: Denis Kristak (Inuits)
Use manual versioning since automatic resolution of the version number fails.
diff -ruN pycistarget-1.0.2_orig/setup.py pycistarget-1.0.2/setup.py
--- pycistarget-1.0.2_orig/setup.py	2022-12-09 12:35:17.000000000 +0000
+++ pycistarget-1.0.2/setup.py	2023-09-25 13:47:20.234303748 +0100
@@ -7,7 +7,7 @@
 
 setup(
      name='pycistarget',
-     use_scm_version=True,
+     version='1.0.2',
      setup_requires=['setuptools_scm'],
      packages=find_packages(),
      include_dirs=["."],
